po/R-zh_CN.po

```diff
@@ -3046,12 +3071,9 @@ msgid "Internal error: factor type mismatch should have been caught earlier"
 msgstr "内部错误：此时不匹配的因子类型应已被发现"
 
 #: shift.R:3
-msgid "Provided argument fill="
-msgstr "提供的 fill= 参数"
-
-#: shift.R:3
-msgid "will be ignored since type='shift'."
-msgstr "将被忽略，因为 type='shift'。"
+#, fuzzy, c-format
+msgid "Provided argument fill=%s will be ignored since type='cyclic'."
+msgstr "将被忽略，因为 type='cyclic'。"
```

Comment:
```suggestion
msgstr "提供的 fill= 参数将被忽略，因为 type='cyclic'。"
```

new message is concatenation of two old messages, did the concatenation manually